identityserver4/src/sql/data/205-Clients-shoppingcart-ui.data.sql

```diff
@@ -0,0 +1,87 @@
+declare @clientId nvarchar(400)
+set @clientId='shoppingcart-ui'
```

Comment:
i might call this shoppingcart-web to match the name of the directory that it's in -- that should also be the service name and docker image name too.